server/utils/middleware.js

```diff
@@ -28,3 +30,20 @@ export function ifNoUser401(req, res, next) {
   }
   return res.status(401).end();
 }
+
+export function sendUserNoEmailNotice() {
```

Comment:
Send this does not have any inputs, it can be a plain middleware instead of a factory that returns a middleware.

Also the name is confusing. Maybe something like `flashIfNotVerified`.